src/components/Header/Pages.js

```diff
@@ -35,6 +38,7 @@ const playerPages = [{
   name: 'Counts',
 }, {
   name: 'Histograms',
+  extra: histogramNames[0],
```

Comment:
One thing I tried earlier was routing to the first item in the histograms if the user went to /histograms, but the problem is that it breaks the back button (since pressing back goes to histograms, then reroutes them to histograms/kills). The solutions I thought of were either to modify back button behavior (which might be tricky if the go to multiple different histograms) or this one, and I thought this one was a bit simpler.